auction-server/src/api/ws.rs

```diff
@@ -287,6 +291,19 @@ impl Subscriber {
         Ok(())
     }
 
+    async fn handle_svm_chain_update(&mut self, svm_chain_update: SvmChainUpdate) -> Result<()> {
+        tracing::Span::current().record("name", "svm_chain_update");
```

Comment:
While we are setting the span name in the `handle_update`, we can safely remove this line.

Reply:
done, should we do the same for `handle_new_opportunity` and `handle_bid_status_update`

Reply:
fixed it for handle_new_opportunity and handle_bid_status_update too